### PR TITLE
QueryRunners: Bind the query function where it is passed on

### PR DESCRIPTION
**Pull request by aocenas**
For some reason the binding wasn't done in a place where the code made the decision to pass on a method as a function which left code implementing the VariableSupport having to know about this and bind the methods to this which is confusing and surprising imho.

### CODE REVIEW

**Review by aangelisc — APPROVED** [review]

**Review by matyax — COMMENTED** [review]
Loki changes look good 👍

**Review by itsmylife — APPROVED** [review]
prom is fine too 👍

**Review by sarahzinger — APPROVED** [review]
Works for cloudwatch!

File: public/app/features/variables/query/queryRunners.test.ts [hunk 1]
```diff
@@ -196,7 +196,7 @@ describe('QueryRunners', () => {
 
       it('then it should call runRequest with correct args', () => {
         expect(runRequest).toHaveBeenCalledTimes(1);
-        expect(runRequest).toHaveBeenCalledWith(datasource, {}, datasource.variables.query);
```
Comment (aocenas) [inline public/app/features/variables/query/queryRunners.test.ts:199]: Not sure about this test change but not sure how meaningful the tests actually are. Seems like they test internal implementation too much and are a bit brittle as this change should not make the tests fail.

**Comment by aocenas** [comment]
@aangelisc @itsmylife @matyax @sarahzinger thanks for the speedy reviews btw, much appreciated :)

**Pull request merged**